components/setting_item_max.jsx

```diff
@@ -271,7 +276,10 @@ export default class SettingItemMax extends React.PureComponent {
             <section
                 className={`section-max form-horizontal ${this.props.containerStyle}`}
             >
-                {title}
+                <div className='d-flex'>
```

Comment:
Does adding this extra div visually change anything for existing settings that don't pass `actionButton`? I noticed some slight spacing changes between the test server and my local instance

https://user-images.githubusercontent.com/3277310/103822326-e6c79a00-503d-11eb-9b3e-7c8099496c2a.mov